boards/sodaq-explorer/Makefile.include

```diff
@@ -0,0 +1,21 @@
+# define the cpu used by the SODAQ ExpLoRer board
+export CPU = samd21
+export CPU_MODEL = samd21j18a
+
+#export needed for flash rule
+export PORT_LINUX ?= /dev/ttyACM0
+export PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+
+# setup serial terminal
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# setup the flash tool used
+# we use BOSSA to flash this board since there's an Arduino bootloader
+# preflashed on it
+export LINKER_SCRIPT ?= $(RIOTCPU)/sam0_common/ldscripts/$(CPU_MODEL)_arduino_bootloader.ld
+include $(RIOTMAKE)/tools/bossa.inc.mk
+
+INCLUDES += -I$(RIOTBOARD)/lorawan-explorer/include
```

Comment:
missed that, replace `/lorawan-explorer/sodaq-explorer/`, or even better with `$(BOARD)` - thought murdock would find such things?

Reply:
actually I think, this isn't needed at all, just for boards that depend on some common includes.